test/run_test.py

```diff
@@ -185,6 +187,181 @@ def __contains__(self, item):
     "distributed/_tensor/test_attention",
 ]
 
+S390X_BLOCKLIST = [
+    # following tests fail for various reasons
```

Comment:
If S390 fails on so many tests, it would perhaps be better to just have a short op-in list in test.sh (similar to how it's currently done for linux-aarch64)

Reply:
For example see https://github.com/pytorch/pytorch/blob/68272ab5967f448ed6d2986039a0ef0ddf0e1b37/test/run_test.py#L1421